### PR TITLE
latestVersion needs to exist

### LINKED ISSUE #99: `package.json` is updated even though `latestVersion` wasn't added
xterm: https://github.com/cdnjs/cdnjs/commit/fa4788c7269db5f77392880ee576de0c661ef17e
Faker: https://github.com/cdnjs/cdnjs/commit/7720617953c4bc07ff27035a3a093b38ffffe1d0

Both of them have a broken autoupdate config ([xterm](https://gist.githubusercontent.com/klausenbusk/12a43115b06878937dba57b2f64cdaf7/raw/2417378c9fb7f4cca5871a228ff21fbfe3468d17/xterm.json), [Faker](https://gist.githubusercontent.com/klausenbusk/12a43115b06878937dba57b2f64cdaf7/raw/2417378c9fb7f4cca5871a228ff21fbfe3468d17/Faker.json))

This probably explains cdnjs/packages#337 and cdnjs/packages#338. I think the issue is caused by #79 

/refs cdnjs/packages#302
/cc @tc80

### PR DESCRIPTION
**Pull request by tc80**
This should fix #99.

### CODE REVIEW

File: cmd/autoupdate/git.go [hunk 1]
```diff
@@ -94,7 +94,7 @@ func updateGit(ctx context.Context, pckg *packages.Package) ([]newVersionToCommi
 		}
 	}
 
-	return newVersionsToCommit, latestGitVersion
+	return newVersionsToCommit, version(lastExistingVersion)
```
Comment (klausenbusk) [inline cmd/autoupdate/git.go:97]: So `package.json` is first updated on the next run?

Comment (tc80) [reply cmd/autoupdate/git.go:97]: it should all be fixed with one run (if my thinking is correct)!

Comment (klausenbusk) [reply cmd/autoupdate/git.go:97]: `lastExistingVersion := git.GetMostRecentExistingVersion(...)` is called before `newVersionsToCommit = doUpdateGit(...)`, so I don't think that is correct?

Comment (klausenbusk) [reply cmd/autoupdate/git.go:97]: Oh, I think you are correct :)

**Pull request merged**